quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -43,8 +56,21 @@
      * @return
      */
     @ResponseBody
-    @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    @RequestMapping(value = "/api/v1/collection/{collectionId}/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @PathVariable String collectionId) {
+        
+        // Every time a new network is submitted, the triple in that network has to be added as a
+        // new MappedTripleGroup for given collectionId
+        MappedTripleGroup mappedTripleGroup;
+        try {
+            mappedTripleGroup = mappedTripleGroupService.addMappedTripleGroup(collectionId);
+            if(mappedTripleGroup == null) {
+                return HttpStatus.NOT_FOUND;
+            }
+        } catch(InvalidObjectIdException | CollectionNotFoundException e)  {
+            logger.error(ExceptionUtils.getStackTrace(e));
```

Comment:
just pass the exception into `error`

Reply:
Since I can't directly do `logger.error(e)` as the type of 'e' is 'Exception', I have to pass e.getMessage() to `error`

Reply:
I believe `error` takes a string and then. an exception.